cfme/tests/test_appliance.py

```diff
@@ -108,15 +109,41 @@ def test_service_enabled(appliance, service):
     assert result.success, result.output
 
 
+@pytest.fixture
+def default_iface(appliance):
+    default_iface = appliance.ssh_client.run_command(
+        "ip r | awk '/^default/ { print $5 }'")
+    return default_iface
+
+
 @pytest.mark.ignore_stream("upstream")
 @pytest.mark.uncollectif(lambda appliance: appliance.is_pod)
-@pytest.mark.parametrize('proto,port', [
-    ('tcp', 22),
-    ('tcp', 80),
-    ('tcp', 443),
-])
-def test_iptables_rules(appliance, proto, port):
-    """Verifies key iptable rules are in place
+def test_firewalld_services_are_active(appliance, default_iface):
+    """Verifies key firewalld services are in place
+
+    Polarion:
+        assignee: jhenner
+        initialEstimate: 1/4h
+        testtype: functional
+        casecomponent: Appliance
+        upstream: no
+    """
+    manageiq_zone = "manageiq"
+    result = appliance.ssh_client.run_command(
+        'firewall-cmd --permanent --zone={} --list-services'.format(manageiq_zone))
+    assert {'ssh', 'http', 'https'} <= set(result.output.split())
+
+    default_iface_zone = appliance.ssh_client.run_command(
+        "firewall-cmd --get-zone-of-interface {}".format(default_iface)
+    ).output.strip()
+    assert default_iface_zone == manageiq_zone
+
+
+@pytest.mark.meta(blockers=[BZ(1712944, forced_streams='5.10')])
```

Comment:
`forced_streams` isn't needed here, and I don't think it should be used. This BZ blocks on 5.10 appliances without it.

```
In [4]: BZ(1712944).blocks
Out[4]: True

In [5]: store.current_appliance.version
Out[5]: Version('5.10.5.0')
```